uint32 Power_ID
time stamp
uint16 Energy_Storage_Voltage
int16 Energy_Storage_Current
uint16 Energy_Storage_Temperature
uint16 other_data
uint16 Fan_Failure
uint16 Fan_State
uint64[] otherdata
uint64[] FanAccident
uint64[] Fandata

